app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  after_save :update_posts_counter
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def update_posts_counter
+    User.find_by(name: author.name).increment!(:posts_counter)
+  end
+
+  def latest_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+end
```

Comment:
- [ ] Same for the `Post` module kindly make sure to include validation here so your app is more secure, the validations should include:
   - [ ] The `title` must not be blank.
   - [ ] The `title` must not exceed 250 characters.
   - [ ] The `comments_counter` must be an integer greater than or equal to zero.
   - [ ] The `likes_counter` must be an integer greater than or equal to zero.